### PR TITLE
New Docusarus site with integrated demo

### PR DESCRIPTION
**Pull request by carloskelly13**
### Description

- Adds new Docusarus site with integrated demo
- Cleans up README and mirrors it in the package so npm shows it
- Migrates to tsup for building
- Removes Cypress and Chromatic since the demo has been decommissioned

### CODE REVIEW

File: packages/nuka/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+[![Nuka Carousel — Formidable, We build the modern web](https://raw.githubusercontent.com/FormidableLabs/nuka-carousel/main/Nuka-Hero.png)](https://formidable.com/open-source/)
```
Comment (gksander) [inline packages/nuka/README.md:1]: For groqd, I pointed this link at the actual groqd docs. Might do the same here but point to nuka's.

File: packages/nuka/package.json [hunk 2]
```diff
@@ -63,15 +58,16 @@
...
     "url-loader": "^4.1.1",
     "webpack": "5.61.0",
     "webpack-cli": "^4.9.1",
     "webpack-dev-server": "^4.6.0"
   },
   "peerDependencies": {
-    "react": "^15.3.0 || ^16.0.0 || ^17.0.1 || ^18.0.0",
-    "react-dom": "^15.3.0 || ^16.0.0 || ^17.0.1 || ^18.0.0"
+    "react": "^18.0.0",
+    "react-dom": "^18.0.0"
```
Comment (gksander) [inline packages/nuka/package.json:70]: Nuka only supports 18 now?

**Review by gksander — APPROVED** [review]
There's a lot here, but looks good to me.

File: packages/nuka/README.md (hunk 1 above)
Comment (carloskelly13) [reply packages/nuka/README.md:1]: Good idea!

**Pull request merged**